lib/kramdown_rpf/rpf.rb

```diff
@@ -140,7 +172,7 @@ def self.convert_quiz_to_html(quiz)
               </h3>
 
               <div class="c-project-quiz__content">
-                #{radio_inputs.join("      ").strip}
+                #{radio_inputs.join('      ').strip}
```

Comment:
rationale? is this a rubocop style thing - I bet @patch0 can point me to the style guide, I just have it ingrained to use double quotes for output strings